remoteworkitem/remoteworkitem_test.go

```diff
@@ -11,8 +10,30 @@ import (
 	"github.com/almighty/almighty-core/test"
 )
 
+const (
+	GithubIssueWithAssignee    = "http://api.github.com/repos/almighty/almighty-core/issues/132"
```

Comment:
More tests need to be added to depict how the code behaves when optional fields like assignee is present.

In fact, the current test cases are too cumbersome to add - and a lot of code is repetitive.
This piece of code you've pointed out would be used for fixing https://github.com/almighty/almighty-core/issues/233  .